airtime_mvc/application/models/Schedule.php

```diff
@@ -156,6 +157,149 @@ public static function GetPlayOrderRangeOld()
         return $range;
     }
 
+    public static function getPVC($utcNow)
```

Comment:
Can you also put comment what this function does, returns, and what the parameter is?